### PR TITLE
Feat/security notice openapi

### PR DESCRIPTION
**Pull request by bracesproul**

### CODE REVIEW

File: docs/docs/modules/agents/toolkits/openapi.md [hunk 1]
```diff
@@ -45,3 +45,9 @@ export const run = async () => {
   );
 };
 ```
+
+# Disclaimer ⚠️
+
+This agent can make requests to external APIs. Use with caution, especially when granting access to users.
+
+Consider adding limitations to what actions can be preformed via the agent, what APIs it can access, what headers can be passed and more.
```
Comment (jacoblee93) [inline docs/docs/modules/agents/toolkits/openapi.md:53]: The potential issue is really that this chain can theoretically send requests with provided credentials/other secrets to hallucinated URLs (thought it should never) - would rephrase around that

**Review by jacoblee93 — COMMENTED** [review]
Small comment

File: docs/docs/modules/agents/toolkits/openapi.md (hunk 1 above)
Comment (bracesproul) [reply docs/docs/modules/agents/toolkits/openapi.md]: Resolved with overall improved language

**Comment by jacoblee93** [comment]
Thanks! We should also add this here:

https://js.langchain.com/docs/modules/chains/popular/api
